site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -74,21 +74,9 @@ The database search engine enables fuzzy search against the following fields of
 
 image::user-search-database.png[User Search with Database Search Engine,width=1200,role=shadowed bottom-cropped]
```

Comment:
Would put another link:

To learn more about the database search engine in general, view the link:...[search core concepts section].